Jenkinsfile

```diff
@@ -1105,14 +1110,34 @@ ton_client/platforms/ton-client-web"""
                         unstash 'web-bin'
                         def deployPath = 'tmp_sdk'
                         if(GIT_BRANCH == "${getVar(G_binversion)}-rc") {
+                            
+                            TON_SDK_BIN_VERSION = GIT_BRANCH.replaceAll(".", "_")
```

Comment:
replaces ALL chars, not only dots

Reply:
Yes, thanks ,confirm the bug in regexp - dot should be double-escaped by backslash to match properly.
Fixed in commits below.